recipe/build.sh

```diff
@@ -1,12 +1,19 @@
 #!/bin/bash
 
-cmake ${CMAKE_ARGS} -DCMAKE_C_COMPILER=$CC -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_INSTALL_LIBDIR=${PREFIX}/lib ${SRC_DIR}
-make install
-
-(cd python
-    cp -r ../src .
-    $PYTHON -m pip install .
-)
-
+mkdir build
 
+cmake "${SRC_DIR}" -B ./build \
+  ${CMAKE_ARGS} \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
+  -DCMAKE_INSTALL_LIBDIR=lib \
+  -DSPGLIB_SHARED_LIBS=ON \
+  -DCMAKE_FIND_FRAMEWORK=NEVER \
+  -DCMAKE_FIND_APPBUNDLE=NEVER \
+  -DSPGLIB_WITH_Fortran=ON
+cmake --build ./build
+# TODO: This should be moved to run_test.sh. Cannot get the appropriate work directory to run these tests
+ctest --test-dir ./build
```

Comment:
How can we move this to `run_test.sh`? I can't find a way to detect the previous work directory?

We would like to use ctest so that it detects all the tests that we support, but we should also be testing outside the build directory to make sure that the installation is functioning.